extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/kotlin/org/jboss/resteasy/reactive/server/runtime/kotlin/CoroutineMethodProcessor.kt

```diff
@@ -0,0 +1,18 @@
+package org.jboss.resteasy.reactive.server.runtime.kotlin
+
+import org.jboss.resteasy.reactive.server.model.HandlerChainCustomizer
+import org.jboss.resteasy.reactive.server.model.ServerResourceMethod
+import org.jboss.resteasy.reactive.server.spi.EndpointInvoker
+import org.jboss.resteasy.reactive.server.spi.ServerRestHandler
+import java.util.function.Supplier
+
+open class CoroutineMethodProcessor(var supplier: Supplier<EndpointInvoker>) : HandlerChainCustomizer {
```

Comment:
You need to put the default constructor back to fix the test failure. I should have left a comment as to why it was there.

Reply:
curious.  I'm still trying to recreate the failure locally (GH actions doesn't seem to want me to view/dl the logs.)  Is there something that tries to instantiate this without a `Supplier`?  I couldn't find any instance of it in the code which is why I cleaned that up.

Reply:
The byte code recording. You can work around it by putting @io.quarkus.runtime.annotations.RecordableConstructor on the constructor, but I did not know how to do that in kotlin.

Reply:
OK.  I'll try that then.  I'd rather avoid the nullable type but in the end, simpler code is better so if nothing else we can just go back to that null default value.  I'll take care of that in a few minutes and get that CI run restarted.

Reply:
ok i reverted that change.  i could've sworn i ran the tests locally but today was a weird day and i clearly didn't.  CI ftw.  They definitely pass now so we should know once CI finishes in a couple of weeks.